src/components/views/elements/OverlaySpinner.js

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
would be nice to use TypeScript for all new code

Reply:
oh shit so i could